core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -74,10 +74,20 @@ trait WorkspaceApiService extends UserInfoDirectives {
           }
         }
       } ~
-      path("workspaces" / Segment / Segment / "migrate") { (workspaceNamespace, workspaceName) =>
+      path("workspaces" / Segment / Segment / "migrate") { (namespace, name) =>
```

Comment:
the API being a `GET  ... migrate` is a bit weird to me. 

I wonder if there's something else we can do? Maybe renaming `migrate` to `migration`?

```suggestion
      path("workspaces" / Segment / Segment / "migration") { (namespace, name) =>
```

then it works with GET and PUT

Reply:
I agree in part but `PUT` migration seems a bit strange too. `POST` might be more appropriate in that case but is `POST` with an empty body ok?

Reply:
throwing out there that we could go with a more verbose option by creating an endpoint for `GET ... migrationStatus`.
I think since this is an undocumented API, sticking to one endpoint might make things easier for us. I think I still prefer `migration` over `migrate`, but I also don't feel super strongly about it.

Reply:
I changed to POST and renamed to `migrations`